src/opm/output/eclipse/Summary.cpp

```diff
@@ -1490,6 +1522,12 @@ static const std::unordered_map< std::string, ofun > funs = {
     { "WNIR", rate< rt::solvent, injector > },
     { "WCIR", rate< rt::polymer, injector > },
     { "WSIR", rate< rt::brine, injector > },
+    { "WTIR#W", ratetracer< rt::tracer, rt::wat, injector > },
```

Comment:
Is the `#` actually keyword syntax? That was new to me.

Reply:
These summary tracer keywords do not discriminate between phases (only implicitly via attached tracer name).  Extending keywords with #W etc allows for phase specific definition of appropriate functions.  There is nothing significant about the use of #, it was chosen simply because the resulting construct will not be confused with actual keywords.

Reply:
OK - sounds reasonable; could maybe add a comment - because all the other keys in this map are actually valid summary keys.